src/Infrastructure/Migrations/Version20230302085340.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\PriceHistoryPlugin\Infrastructure\Migrations;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230302085340 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Populates the sylius_price_history_channel_pricing_log_entry table with initial data';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->abortIf(
+            !$this->getSchemaManager($this->connection)->tablesExist(['sylius_price_history_channel_pricing_log_entry']),
+            'The table sylius_price_history_channel_pricing_log_entry does not exist.',
+        );
```

Comment:
What could be the case that the table does not exist?

Reply:
I decided to make sure we have a table we're going to operate on. It might be redundant, I can delete it if anybody more raise this issue :p.

Reply:
+1 for removal

Reply:
What is more, I would be for adding these inserts into migration with the creation of `sylius_price_history_channel_pricing_log_entry` table as this plugin is not even tagged and then this check will not be needed.

Reply:
Ok, I did it the @GSadee way.